### DISCUSSION TITLE
Back up to designated album

### DISCUSSION THREAD

**Discussion by sliencio** [immich-app/immich#14240] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

First of all, I really like immich, which is such an easy-to-use and powerful software service. However, I encountered a small problem when backing up my iPhone. Because I built the project using nas, immich hosts the backup of multiple people's mobile phones, and everything is synchronized on the iPhone. It is a recent project, so all the pictures imported to immich will be in the "Recent Projects" album. It will appear confusing. Is it possible to add the function of backing up to immich's designated album during backup? Thank you so much! ! !

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
What do you mean by "Immich's designated album"?
